webapp/src/components/Wallet/AccountStatus.tsx

```diff
@@ -500,8 +510,13 @@ const AccountStatus: React.FC<AccountStatusProps> = ({
               handleCopyAddress,
               renderCopiedButton()
             )}
-            {chainId && renderMenuItem(`OPEN IN ${BLOCKCHAIN_EXPLORER_NAME[chainId]}`, handleOpenEtherscan)}
+            {chainId &&
+              renderMenuItem(
+                `OPEN IN ${BLOCKCHAIN_EXPLORER_NAME[chainId]}`,
+                handleOpenEtherscan
+              )}
             {renderMenuItem("DISCONNECT", handleDisconnect)}
+            {renderAirdropButton()}
```

Comment:
FYI, These really should be a component and declared like `<AirdropBotton />` Inline rendering components like this causes a performance hit because the virtual DOM can't follow the reconciliation process and screws up the render tree.

https://stackoverflow.com/questions/67454966/whats-the-difference-between-functions-that-render-jsx-vs-declaring-components